pypesto/sample/parallel_tempering.py

```diff
@@ -184,14 +184,21 @@ def compute_log_evidence(
         method: str = "trapezoid",
         use_all_chains: bool = True,
     ) -> Union[float, None]:
-        """Perform thermodynamic integration to estimate the log evidence.
+        """Perform thermodynamic integration or steppingstone sampling to estimate the log evidence.
```

Comment:
This belongs in `sample/estimate_evidence.py`.

As far as I can tell, `self.betas` is stored in `result` anyway. The only benefit of having it here is to emit a warning about a suboptimal temperature schedule. Store `beta_init` in the result or move the advice to the docstring(s) of `ParallelTemperingSampler.options["beta_init"]`.